src/SDPoptimize.jl

```diff
@@ -465,13 +465,15 @@ Parameters:
 
 Returns :
 - V(x0) (Float64)
+
 """
-function get_value(model::SPModel,param::SDPparameters,V::Array{Float64})
+function get_bellman_value(model::SPModel, param::SDPparameters, V::Array{Float64})
```

Comment:
get_bellman_value refers intuitively to a function that return the value of the Bellman function, whereas this function return only V_0(x0) (to be changed in l467 above). I suggest get_optimal_value or something like that instead.